packages/gatsby/src/query/better-queue-custom-store.ts

```diff
@@ -1,32 +1,40 @@
-function MemoryStoreWithPriorityBuckets() {
+import { Store } from "better-queue"
+
+// getRunningTasks is an extension to the interface, and is used in the tests
+interface IGatsbyBetterStore<T> extends Store<T> {
```

Comment:
Obviously, this goes against the guidelines of not using Generics, but if there's a good type for the Tasks then this can be updated.